src/test/java/org/fundacionjala/coding/yerel/PlanetTest.java

```diff
@@ -0,0 +1,66 @@
+package org.fundacionjala.coding.yerel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+
+/**
+ * this is a Junit planet.
+ */
+public class PlanetTest {
+    private static final int NUMBER_ONE = 1;
+    private static final int NUMBER_TWO = 2;
+    private static final int NUMBER_THREE = 3;
+    private static final int NUMBER_FOUR = 4;
+    private static final int NUMBER_FIVE = 5;
+    private static final int NUMBER_SIX = 6;
+    private static final int NUMBER_SEVEN = 7;
+    private static final int NUMBER_EIGTH = 8;
+    private static final int NUMBER_NINE = 9;
+    private static final int NUMBER_TWELVEN = 12;
+    private static final int NUMBER_THIRTEEN = 13;
+    private static final int NUMBER_FOURTEEN = 14;
```

Comment:
Don't initialize this numbers as constants
You should declare the arrays as final